### PR TITLE
fix: venues search variables

### PR DESCRIPTION
**Pull request by melniiv**
### Closes

**[DEV-XXX](https://helsinkisolutionoffice.atlassian.net/browse/DEV-XXX):**

### CODE REVIEW

**Comment by terovirtanen** [comment]
**Sports-Helsinki branch is deployed to platta: https://liikunta-106-liikunta-346-search.dev.hel.ninja** :rocket::rocket::rocket:

**Review by nikomakela — APPROVED** [review]

**Comment by tuomas777** [comment]
Lines 92-95 in the same file try to also set those default values (but apparently fail miserably), so to make that less confusing those defaults could be removed.

**Comment by melniiv** [comment]
yes, thanks, i thought i removed the default vars.

File: apps/sports-helsinki/src/domain/unifiedSearch/useUnifiedSearchVariables.ts [hunk 1]
```diff
@@ -90,9 +90,9 @@ export default function useUnifiedSearchVariables(
     filters: {
       q,
       // By default filter by the sports dept. ontology tree id
-      ontologyTreeIds = [SPORTS_DEPARTMENT_ONTOLOGY_TREE_ID],
+      ontologyTreeIds,
       // Limit results inside Helsinki when there is no administrative division(s) selected
```
Comment (tuomas777) [inline apps/sports-helsinki/src/domain/unifiedSearch/useUnifiedSearchVariables.ts:94]: I think these comments should be now removed from here as well 🙂

Comment (melniiv) [reply apps/sports-helsinki/src/domain/unifiedSearch/useUnifiedSearchVariables.ts]: moved to proper place, those comments are still important to have

**Review by tuomas777 — APPROVED** [review]
👍

**Pull request merged**